runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -116,6 +119,10 @@
     @Getter
     private final CorfuTable<ProtobufFileName, CorfuRecord<ProtobufFileDescriptor, TableMetadata>> protobufDescriptorTable;
 
+    @Getter
+    private final CorfuTable<TableName, CorfuStoreMetadata.KeyValueDefaultRecord>
```

Comment:
TableRegistry already maps tableName to a list of protos, instead of a separate table, can we instead just put a boolean field or metadata in the TableRegistry's descriptor to indicate who is the default protobuf file for a given tableName? This will otherwise double the memory for the TableRegistry's types, which can be expensive.

Reply:
it is not sufficient to get the file name or file descriptor proto.  We need to somehow construct a dynamic message of the required type using which the key and value json strings can be parsed.  I couldnt find any way to do that using the file descriptor proto or file name.

Reply:
This will increase the memory used for the dynamic serializer for all consumers while it's only needed for the browser tool. 

Essentially, you need to map a Table -> fileNames -> descriptors. I though the table metadata contained which types are used in a table. No?

Reply:
instead of the separate table can we just add the new fields into the TableDescriptors message itself? This has an advantage of serving as a schema registry in just one table. It will also simplify a lot of queries in the future where the schema needs to be looked up?